web/src/app/admin/admin-outgoing-logs/OutgoingLogCard.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import {
+  CardContent,
+  CardHeader,
+  Typography,
+  Card,
+  CardActions,
+  Chip,
+} from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles'
+import { DateTime } from 'luxon'
+import { DebugMessage } from '../../../schema'
+import toTitleCase from '../../util/toTitleCase'
+import { theme } from '../../mui'
+
+const useStyles = makeStyles<typeof theme>((theme) => ({
+  card: {
+    marginTop: theme.spacing(0.5),
+    marginBottom: theme.spacing(0.5),
+    cursor: 'pointer',
+  },
+  chip: {
+    padding: '2px 8px',
+    borderRadius: '15px',
+  },
+  msgType: {
+    fontSize: 14,
+  },
+}))
+
+interface Props {
+  debugMessage: DebugMessage
+  selected: boolean
+  onSelect: () => void
+}
+
+export default function OutgoingLogCard(props: Props): JSX.Element {
+  const { debugMessage, selected, onSelect } = props
+  const classes = useStyles()
+
+  const type = debugMessage.type
+  const status = toTitleCase(debugMessage.status)
+  const statusDict = {
+    success: {
+      backgroundColor: '#EDF6ED',
+      color: '#1D4620',
+    },
+    error: {
+      backgroundColor: '#FDEBE9',
+      color: '#611A15',
+    },
+    warning: {
+      backgroundColor: '#FFF4E5',
+      color: '#663C00',
+    },
+    info: {
+      backgroundColor: '#E8F4FD',
+      color: '#0E3C61',
+    },
+  }
+
+  let statusStyles
+  const s = status.toLowerCase()
+  if (s.includes('deliver')) statusStyles = statusDict.success
```

Comment:
This can include `sent` which is also a final-state -- delivery notification isn't always supported

Reply:
Added 👍